server/webserver.go

```diff
@@ -740,23 +740,13 @@ func (s *LivepeerServer) StartWebserver(bindAddr string) {
 	mux.HandleFunc("/status", func(w http.ResponseWriter, r *http.Request) {
 		status := s.GetNodeStatus()
 		if status != nil {
-			mstrs := make(map[string]string, 0)
-			for mid, m := range status.Manifests {
-				mstrs[string(mid)] = m.String()
-			}
-			d := struct {
-				Manifests map[string]string
-				Version   string
-			}{
-				Manifests: mstrs,
-				Version:   core.LivepeerVersion,
-			}
-			if data, err := json.Marshal(d); err == nil {
+			if data, err := json.Marshal(status); err == nil {
```

Comment:
Should we start doing expects-style testing for this?

Reply:
What do you mean by that?

Reply:
Mostly checking the JSON output of the `/status` call. Roughly paraphrased:

```
w *ResponseWriter
r *Request
err = test_call('/status', w, r)
expected := `{"manifests": ..., "version": ... }` 
body := w.Body.ReadAll()
if body != expected {
  t.Error("Body did not match; expected '%s 'got '%s'", expected, body)
}  
```

Especially since the `GetNodeStatus` call seems to have a lot of possible variations (orchestrators, registered transcoders, etc). Alternatively, we could have more involved tests for `GetNodeStatus` and a simpler test for this `/status` endpoint.

(The CLI API is lacking test coverage; improving that as we make CLI changes seems like a good opportunity.)

Reply:
Added one simple test for `/status` endpoint